interview-questions.md

```diff
@@ -7,41 +7,41 @@ Then, research the question to expand on your answer. Even if you feel you have
 
```

Comment:
Hand clap. Just to further enhance this great approach to interview questions. Share examples that you help your cohort understand these topics, tell any analogies that you used, or how you related to a previous subject. You can even drop that "while volunteering as a Teacher Assistant, I...." Let these people know how awesome and considerate you are.